### PR TITLE
Adaptivity

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: interface-elements-for-web/articles/card-view.md [hunk 1]
```diff
@@ -23,5 +23,6 @@ This section describes the capabilities provided by the Card View control, which
```
Comment (konstantinbelyakov) [inline interface-elements-for-web/articles/card-view.md]: @AnnaKondratova it seems that a link to the "Exporting" section is missing here (and in toc.yaml too).

Comment (AnnaKondratova) [reply interface-elements-for-web/articles/card-view.md]: Modified.

File: interface-elements-for-web/articles/file-manager/adaptive-layout.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+title: Adaptive Layout
+author: Anna Kondratova
+---
+# Adaptive Layout
+
+The file manager's elements are automatically rearranged when the browser window is resized. 
+
+## Folder Container
+
+Click the hamburger button to expand/collapse the folder container in the toolbar's left.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/file-manager/adaptive-layout.md]: "... in the toolbar's left." left what?

Comment (AnnaKondratova) [reply interface-elements-for-web/articles/file-manager/adaptive-layout.md]: left side

File: interface-elements-for-web/articles/card-view.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ author: Svetlana Nikulina
 legacyId: 115295
 ---
 # Card View
-This section describes the capabilities provided by the Card View control, which represents data in cards, supports data editing, sorting, filtering, summary calculation, and many other features.
+This section describes the Card View control's capabilities. It represents data in cards, supports data editing, sorting, filtering, summary calculation, and many other features.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/card-view.md]: "... and many other features." This is too vague.

File: interface-elements-for-web/articles/grid.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ author: Svetlana Nikulina
 legacyId: 4854
 ---
 # Grid
-This section describes the capabilities provided by the Grid control, which represents data in a tabular form, supports data editing, sorting, grouping, filtering, summary calculation and many other features.
+This section describes the Grid control's capabilities. It represents data in a tabular form, supports data editing, sorting, grouping, filtering, summary calculation and many other features.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/grid.md]: "... and many other features." This is too vague.

File: interface-elements-for-web/articles/menu.md [hunk 5]
```diff
@@ -4,11 +4,12 @@ author: Svetlana Nikulina
 legacyId: 9382
 ---
 # Menu
-Menu is used to navigate you through items.
+The menu is used to navigate through items.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/menu.md]: "The menu is used to navigate through items." I'm not sure this is the best description of what the 'Menu' is.

File: interface-elements-for-web/articles/menu/menu-scrolling/menu-scrolling.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: Menu Scrolling
+author: Svetlana Nikulina
+legacyId: 9383
+---
+# Menu Scrolling
+A submenu displays scroll buttons if it does not fit into the visible document area. To scroll through a submenu use the button.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/menu/menu-scrolling/menu-scrolling.md]: "To scroll through a submenu use the button." Which button?

File: interface-elements-for-web/articles/card-view.md (hunk 3 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/card-view.md]: Modified.

File: interface-elements-for-web/articles/menu/responsive-layout/responsive-layout.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+title: Responsive Layout
+author: Anna Kondratova
+---
+# Responsive Layout
+
+The menu can operate in the following modes when the browser window's width changes:
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/menu/responsive-layout/responsive-layout.md]: The two sentences do not read like they are 'modes'.

File: interface-elements-for-web/articles/grid.md (hunk 4 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/grid.md]: Modified.

File: interface-elements-for-web/articles/menu.md (hunk 5 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/menu.md]: Modified.

File: interface-elements-for-web/articles/menu/menu-scrolling/menu-scrolling.md (hunk 6 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/menu/menu-scrolling/menu-scrolling.md]: Updated.

File: interface-elements-for-web/articles/menu/responsive-layout/responsive-layout.md (hunk 7 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/menu/responsive-layout/responsive-layout.md]: I still think it's the modes :) We can switch on/off each of these modes. Maybe the word "way" is better here?

**Pull request merged**